docs/docs/contributing/contribute-component.md

```diff
@@ -0,0 +1,20 @@
+# How to contribute components?
+
+As of Langflow 1.0 alpha, new components are added as objects of the [CustomComponent](https://github.com/langflow-ai/langflow/blob/dev/src/backend/base/langflow/interface/custom/custom_component/custom_component.py) class and any dependencies are added to the [pyproject.toml](https://github.com/langflow-ai/langflow/blob/dev/pyproject.toml#L27) file.
+
+## Add an example component
+
+You have a new document loader called **MyCustomEmbedding** and it would look awesome in Langflow.
```

Comment:
Hey @mendonk 

The only observation would be to change the name of the new component to something related to doument_loaders because you are adding the new component to that folder